stock_view_adj_hls/views/stock_quant_views.xml

```diff
@@ -25,4 +25,16 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_stock_move_line_operation_tree" model="ir.ui.view">
+        <field name="name">stock.move.line.operations.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id"
+               ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <field name="lot_id" position="attributes">
```

Comment:
@TomohiroAbe-Quartile Actually, why `lot_id`? Shouldn't we be adjusting `lot_name`?

Reply:
@yostashiro 
I edit this field.
![image](https://user-images.githubusercontent.com/84064473/128132817-4607d088-0f08-41b8-a138-9acc7ea8610d.png)

Reply:
![image](https://user-images.githubusercontent.com/84064473/128146088-d591788c-96b5-4987-accf-d69dee854ba8.png)

I confirmed lot_name field and add invisible attribute